onnxruntime/core/graph/contrib_ops/nhwc_schema_defs.cc

```diff
@@ -90,6 +90,10 @@ class NhwcInferenceContext : public InferenceContext {
     return nullptr;
   }
 
+  const SparseTensorProto* getInputSparseData(size_t) const override {
```

Comment:
just checking: do we need all these changes in this PR to keep the CIs happy ?

Reply:
Yes, this a minor part of Dmitri's PR for sparse tensors.